### PR TITLE
fix: issues while build

### PR DESCRIPTION
**Pull request by neoddish**
### PR includes

- [x] fixed 
- [x] add / modify test cases

### CODE REVIEW

File: packages/ava/src/data/statistics/index.ts [hunk 1]
```diff
@@ -252,10 +252,9 @@ export function missing(value: unknown[]): number {
 export function valueMap(value: unknown[]): Record<string, number> {
   const data: Record<string | number, number> = {};
   value.forEach((v) => {
-    if (isString(v) || isNumber(v)) {
-      if (data[v]) data[v] += 1;
-      else data[v] = 1;
-    }
+    const index = String(v);
```
Comment (neoddish) [inline packages/ava/src/data/statistics/index.ts:255]: 好像 index 改成 key 更好。。

File: packages/ava/src/data/statistics/index.ts [hunk 2]
```diff
@@ -252,10 +252,9 @@ export function missing(value: unknown[]): number {
 export function valueMap(value: unknown[]): Record<string, number> {
   const data: Record<string | number, number> = {};
   value.forEach((v) => {
-    if (isString(v) || isNumber(v)) {
-      if (data[v]) data[v] += 1;
-      else data[v] = 1;
-    }
+    const key = String(v);
```
Comment (pddpd) [inline packages/ava/src/data/statistics/index.ts:255]: it will conflict with https://github.com/antvis/AVA/pull/492

**Review by pddpd — CHANGES_REQUESTED** [review]
the type of commit msg should be modified to `fix(ava)`

**Review by pddpd — APPROVED** [review]
LGTM

**Pull request merged**